led_strip/examples/led_strip_rmt_ws2812/main/idf_component.yml

```diff
@@ -3,3 +3,6 @@ dependencies:
   espressif/led_strip:
     version: '^2'
     override_path: '../../../'
+
+  led_strip:
+    path: /home/chenjichang/task/idf-extra-components/led_strip
```

Comment:
Let's simplify the examples by creating a new example: `led_strip/examples/led_strip_spi_ws2812`. This line can be removed